lib/events/uploader.go

```diff
@@ -219,7 +219,7 @@ func (u *Uploader) emitEvent(e UploadEvent) {
 }
 
 func (u *Uploader) uploadFile(lockFilePath string, sessionID session.ID) error {
-	lockFile, err := os.Open(lockFilePath)
+	lockFile, err := os.OpenFile(lockFilePath, os.O_RDWR, 0640)
```

Comment:
Nit: Cleaner to just pass `0` for permission bits since we aren't using `O_CREATE`.

Reply:
@fspmarshall good point, fixed 👍